src/main/java/com/michelin/ns4kafka/controller/StreamController.java

```diff
@@ -197,7 +200,11 @@ HttpResponse<List<KafkaStream>> bulkDelete(
         kafkaStreams.forEach(kafkaStream -> {
             sendEventLog(kafkaStream, ApplyStatus.DELETED, kafkaStream.getMetadata(), null, EMPTY_STRING);
 
-            streamService.delete(ns, kafkaStream);
+            try {
```

Comment:
Let's just move to a traditional iteration here and add exceptions to signature:

```java
for (KafkaStream kafkaStream : kafkaStreams) {
    sendEventLog(kafkaStream, ApplyStatus.DELETED, kafkaStream.getMetadata(), null, EMPTY_STRING);
    streamService.delete(ns, kafkaStream);
}
```